packages/react-reconciler/src/ReactFiberCommitWork.js

```diff
@@ -1090,6 +1158,19 @@ function commitPlacement(finishedWork: Fiber): void {
       // down its children. Instead, we'll get insertions from each child in
       // the portal directly.
     } else if (node.child !== null) {
+      if (
+        enableDoNotUseGetHostNodes &&
+        node.tag === SuspenseComponent &&
+        node.memoizedProps !== null &&
+        node.memoizedState === null &&
+        typeof node.memoizedProps.unstable_do_not_use_getHostNodes ===
+          'function'
+      ) {
```

Comment:
I don't get why any of this new code is necessary.

Reply:
Oh this is to handle the initial render, but not in the place I expected. Let's handle this in `appendAllChildren` in the complete phase instead, since we already do a traversal there.